### DISCUSSION TITLE
Flash session not displaying after redirect

### DISCUSSION THREAD

**Discussion by Shonetow** [livewire/livewire#6861] [7. Bugs (archived)]
The flash session that's set within the method invoked upon submit (or by click) doesn't display after the redirect.

```php
public function handleEmailSubmit(): void
    {
        session()->flash('old_email', 'john@exmaple.com');
        // dd(session('old_email')); // it works
        $this->redirectRoute('login');
        // return redirect()->to('login'); // tried, not working
    }
```

However, if I place the same code in the Livewire's `boot()` method, it works as expected.

Both **register** and **login** pages are **Blade components**, while the register page contains Livewire component where the logic is planted.

Strange, as it worked when I first set it up, but after some time it stopped, without me changing the configuration or anything that would logically affect this malfunction.

I've tried many code modifications to figure out the issue before deciding to post an issue here.

Any ideas please?

**Comment by JSFTI** [comment]
I think nesting livewire components break flash session. I removed all livewire components from my view and flash sessions works again.
